recipes/acpia-tools/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+context:
+  version: "2025_04_04"
+
+package:
+  name: acpica-tools
+  version: ${{ version | replace('_', '') }}
+
+source:
+  url: https://github.com/acpica/acpica/archive/refs/tags/R${{ version }}.tar.gz
+  sha256: 9991ec103b3660d17715780406ee7409f705cf87ac55e9a32374affe1a6f275a
+
+build:
+  number: 0
+  script: |
+    make PREFIX=$PREFIX OPT_CFLAGS="-DACPI_PACKED_POINTERS_NOT_SUPPORTED"
+    make install PREFIX=$PREFIX
+  skip: not unix
```

Comment:
Windows support requires Cygwin and dependencies are also missing: https://github.com/acpica/acpica/blob/cbc243e46f577425838ef61a88d00e118a8928ef/generate/release/build.txt#L43-L56